Setup Claude (#67)
LiskHQ/panna-sdk: .claude/agents/git-branch-reviewer.md

tools: Glob, Grep, Read, WebFetch, TodoWrite, WebSearch, BashOutput, KillShell
model: sonnet
---

> 

Author: Done.

---
name: git-branch-reviewer
description: Use this agent when you need to review code changes on the current Git branch before creating or finalizing a pull request. This agent performs preliminary code review by analyzing diffs, identifying potential issues, and suggesting improvements.

> tools: Glob, Grep, Read, WebFetch, TodoWrite, WebSearch, BashOutput, KillShell

Reviewer: If we also add an `Edit` tool to the agent, it could optionally suggest fixes directly in code (with user approval).

> tools: Glob, Grep, Read, WebFetch, TodoWrite, WebSearch, BashOutput, KillShell

Author: Done.

model: sonnet
examples:

- context: The user has made several commits on a feature branch and wants a review before opening a PR.

  user: "I've finished implementing the new authentication feature, can you review my changes?"
  assistant: "I'll use the git-branch-reviewer agent to analyze the changes on your current branch."
  commentary: |
  Since the user has completed work on a feature and wants a review before the PR, use the git-branch-reviewer agent to examine the branch changes.

- context: The user wants to ensure their branch changes meet quality standards.
  user: "Check if my branch is ready for PR"
  assistant: "Let me launch the git-branch-reviewer agent to perform a preliminary review of your branch changes."
  commentary: |
  The user is asking for a pre-PR review, so use the git-branch-reviewer agent to analyze the current branch.
---

> You are an expert code reviewer specializing in pre-pull request analysis. Your role is to review changes on the current Git branch compared to the base branch (typically development but occasionally main or master) and provide actionable feedback before a pull request is created.

Reviewer: We don't use `master` branch?

> You are an expert code reviewer specializing in pre-pull request analysis. Your role is to review changes on the current Git branch compared to the base branch (typically development but occasionally main or master) and provide actionable feedback before a pull request is created.

Author: Fixed.

> You are an expert code reviewer specializing in pre-pull request analysis. Your role is to review changes on the current Git branch compared to the base branch (typically development but occasionally main) and provide actionable feedback before a pull request is created.

Reviewer: ```suggestion
You are an expert code reviewer specializing in pre-pull request analysis. Your role is to review changes on the current Git branch compared to the base branch (typically the repository's default branch - in this case 'development') and provide actionable feedback before a pull request is created.
```
